### DISCUSSION TITLE
Name of mortgage feature branch

### DISCUSSION THREAD

**Discussion by DanyaoWang-001167576** [ian-tiggeman/T24S2T1EXTG08#5] [Q&A] [answered]
Hi Ian,
I know we talked about this a few weeks ago at the Oral Communication Workshop. I just want to double check that the name of the mortgage feature branch is **feature_mortgage**?
Thanks,
Cara

**Comment by ian-tiggeman** [comment, marked as answer]
Hi Cara,
Yes, your feature branch  being called **feature_mortgage** is correct.
Good job.
Regards,
Ian
